mdoc/Test/en.expected/Mono.DocTest.Generic/GenericBase`1.xml

```diff
@@ -127,7 +127,7 @@
       <MemberSignature Language="VB.NET" Value="Public Custom Event MyEvent As EventHandler(Of GenericBase(Of U).FooEventArgs) " />
       <MemberSignature Language="F#" Value="member this.MyEvent : EventHandler&lt;Mono.DocTest.Generic.GenericBase&lt;'U&gt;.FooEventArgs&gt; " Usage="member this.MyEvent : System.EventHandler&lt;Mono.DocTest.Generic.GenericBase&lt;'U&gt;.FooEventArgs&gt; " />
       <MemberSignature Language="C++ CLI" Value="public:&#xA; event EventHandler&lt;Mono::DocTest::Generic::GenericBase&lt;U&gt;::FooEventArgs ^&gt; ^ MyEvent;" />
-      <MemberSignature Language="C++ WINRT" Value="// Register&#xA;event_token MyEvent(EventHandler&lt;Mono::DocTest::Generic::GenericBase&lt;U&gt;::FooEventArgs&gt; const&amp; handler) const;&#xA;&#xA;// Revoke with event_token&#xA;void MyEvent(event_token const* cookie) const;&#xA;&#xA;// Revoke with event_revoker&#xA;MyEvent_revoker MyEvent(auto_revoke_t, EventHandler&lt;Mono::DocTest::Generic::GenericBase&lt;U&gt;::FooEventArgs&gt; const&amp; handler) const;" />
+      <MemberSignature Language="C++ WINRT" Value="// Register&#xA;event_token MyEvent(EventHandler&lt;Mono::DocTest::Generic::GenericBase&lt;U&gt;::FooEventArgs&gt; const&amp; handler) const;&#xA;&#xA;// Revoke with event_token&#xA;void MyEvent(event_token const* cookie) const;&#xA;&#xA;// Revoke with event_revoker&#xA;GenericBase`1::MyEvent_revoker MyEvent(auto_revoke_t, EventHandler&lt;Mono::DocTest::Generic::GenericBase&lt;U&gt;::FooEventArgs&gt; const&amp; handler) const;" />
```

Comment:
Looks like you didn't convert the generic type `GenericBase`1::` to a correct type name.

Reply:
Thanks @hiihellox10 . Referring to [DocumentationEnumerator.GetDocumentationTypes() method](https://github.com/mono/api-doc-tools/blob/24ba59fcca3d1acacdb0f71acbdf401d5ce3fb96/mdoc/Mono.Documentation/Updater/DocumentationEnumerator.cs#L17), it seems to be good. The recognized type is "Mono.DocTest.Generic.GenericBase`1"